components/src/core/components/InputField/input-field.scss

```diff
@@ -13,3 +13,7 @@
   content: '*';
   color: $oxd-input-field-required-asterisk-color;
 }
+
+.orangehrm-input-hint {
+  font-size: 0.75rem;
+}
```

Comment:
line end and 0.75rem can move to variable, if any adjustment is needed from app side

Reply:
I moved the font-size to a variable, and added the other basic font styles also here. (weight, color, family)